packages/website/src/components/CurriculumSection/CurriculumSection.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```

Comment:
💭 To follow our conventions you can replace all `CurriculumSection` file/folder names with just `Curriculum`